src/v2/ion/responseTransformer.js

```diff
@@ -197,6 +197,27 @@ const modifyFormNameForIdPAuthenticator = result => {
 
 };
 
+/**
+ * API reuses `redirect-idp` remeditaion form for PIV IdP.
+ * However the UX for PIV IdP is different from social idps in terms of the PIV
+ * instructions view that needs to be rendered before we redirect to mtls.
+ *
+ * This function changes `redirect-idp` to `piv-idp` for PIV IdP.
+ */
+const modifyFormNameForPIVIdP = result => {
+  if (Array.isArray(result.remediations)) {
+    const pivIdp = result.remediations.filter(
+      remediation => {
+        return remediation.name === RemediationForms.REDIRECT_IDP
```

Comment:
nit: we don't need return and {}.

Reply:
please try to generalize `modifyFormNameForIdPAuthenticator` instead of duplicating.